bundles/org.openhab.binding.resol/README.md

```diff
@@ -0,0 +1,250 @@
+ * Frista
+ * DeltaTherm® HC
+ * DeltaTherm® FK
+ * Deltatherm® HT
+ * DeltaTherm® DH
+ * Sonnenkraft SKSC3
+ * Sonnenkraft STRG BX PLUS
+ * Sonnenkraft SKSC3+
+ * Sonnenkraft SKSC3HE
+ * Sonnenkraft SKSR 1/2/3
+ * Vitosolic 200
+ * COSMO Multi
+ * Drainback DeDietrich
+ * Diemasol C
+
+A more complete list can be found in the doc of the [resol-vbus-java library](http://danielwippermann.github.io/resol-vbus/vbus-packets.html).
+
+###Emulated Extension Module EM
+
+Some controllers like the Deltasol MX can be connected to an extension module, which can be emulated by the thing type *emulatedEM*.
+The emulated EM is a virtual device, visible on the VBUS to a Resol controller and provides an interface between openHAB and the controller.
+Relay channels are outputs from the controller point of view and therefore read-only in OH.
+The sensor channels as inputs for the solar or system controller and intended to be written by OH.
+
+
+## Discovery
+
+Discovery is tested for VBus-LAN adapters DL2, DL3 and KM2 devices, it should also work for other devices providing a live data port.
+After a bridge is detected in the local network the password needs to be given and the things on the VBUS will popup in the inbox.
+
+
+## Bridge Configuration
+
+The bridge is the device connecting the Resol VBUS to the network, usually a VBus-LAN adapter or integrated in some of the solar controllers like DL3.
+For the connection from the Resol binding the bridge requires the configuration of
+
+| property             | type    | Required | description                                                |
+|----------------------|---------|----------|------------------------------------------------------------|
+| ipAddress            | String  | yes      | IP address or hostname of the VBUS adapter                 |
+| password             | String  | yes      | Password, defaults to 'vbus' for factory setting devices   |
+| port                 | Number  | no       | Port for the TCP connection, defaults to 7053              |
+| adapterSerial        | String  | no       | Serialnumber of the device (informative only)              |
+
+## Device Configuration
+
+Depending on the solar/heating controller you have attached to your VBUS there will be a "controller" and several other things like heat quantity meters, heating circuit controls, etc.
+These do not require any configuration parameters and will pop up in your inbox after the bridge has received data from them.
+The name of the devices is usually the Resol name with spaced replaced by _ and a "-Controller" suffix like "DeltaSol_MX-Controller".
+For configuration in files you can enable the logging with at least DEBUG level for the resol binding and search the logs for "ThingHandler for (.*) not registered." to identify the names of the things you can add for your VBUS devices.
+
+## Emulated EM Configuration
+
+*emulatedEM* devices cannot be auto-discovered and require beside the bridge the following configuration:
```

Comment:
```suggestion
*emulatedEM* devices cannot be auto-discovered and require beside the bridge the following configuration:

```

Reply:
What is different here? I am too blind to see the change from "and" to "and" :cry:

Reply:
It's the new line after the colon. There should be an empty line before tables, head lines and so on.

Reply:
ah yes, clear. I was focused on the red indicated and in the diff such that I didn't recognize the newline, sorry.